nomos-da/network/core/src/swarm/common/mod.rs

```diff
@@ -1,4 +1,14 @@
+use std::time::Duration;
+
+use serde::{Deserialize, Serialize};
+
 pub mod balancer;
 pub mod handlers;
 pub mod monitor;
 pub mod policy;
+
+#[derive(Copy, Clone, Debug, Default, Serialize, Deserialize)]
+pub struct DAReplicationSettings {
+    pub seen_message_cache_size: usize,
+    pub seen_message_ttl: Duration,
+}
```

Comment:
I don't get why we have two of these for the same attributes.

Reply:
Since this is used by replication behavior, seems better to define in behavior and then reexport in swarm common. The idea would be similar to `ReplicationEvent`, we have it defined in the behavior too. Thats how policy and monitor settings are organized also.